maven-embedder/src/main/java/org/apache/maven/cli/logging/impl/JDKConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.maven.cli.logging.impl;
+
+import java.util.logging.Logger;
+
+import org.apache.maven.cli.logging.BaseSlf4jConfiguration;
+
+/**
+ * Configuration for slf4j-jdk14.
+ */
+public class JDKConfiguration extends BaseSlf4jConfiguration {
```

Comment:
This is additional API I'd rather not carry around. I need to understand the root issue better, but it still seems to me either warning is correct, in which case we should do nothing, or we should remove the log message completely. I don't like half measures and work arounds for misconfigured builds.

Reply:
well, this one is easy, maven integrates with logback, log4j2, etc so it integrated with slf4j bindings, jdk14 is just a standard one so we must integrate with, no real ambiguity.
now if I didn't get it and you mean maven shouldn't integrate with slf4j binding - I can agree with that at some point - then I'd be happy to drop logback, log4j2 and other integrations too.

Reply:
OK, as long as this means JDK 1.4 and not JDK 14.0 no problem here. Time marches on...